diff --git a/src/ChalkBox/__pycache__/__init__.cpython-36.pyc b/src/ChalkBox/__pycache__/__init__.cpython-36.pyc
index 9db9765..0a4486a 100644
Binary files a/src/ChalkBox/__pycache__/__init__.cpython-36.pyc and b/src/ChalkBox/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/ChalkBox/__pycache__/settings.cpython-36.pyc b/src/ChalkBox/__pycache__/settings.cpython-36.pyc
index c862d3e..b1d86f8 100644
Binary files a/src/ChalkBox/__pycache__/settings.cpython-36.pyc and b/src/ChalkBox/__pycache__/settings.cpython-36.pyc differ
diff --git a/src/ChalkBox/__pycache__/urls.cpython-36.pyc b/src/ChalkBox/__pycache__/urls.cpython-36.pyc
index edc5c53..c77a300 100644
Binary files a/src/ChalkBox/__pycache__/urls.cpython-36.pyc and b/src/ChalkBox/__pycache__/urls.cpython-36.pyc differ
diff --git a/src/ChalkBox/__pycache__/views.cpython-36.pyc b/src/ChalkBox/__pycache__/views.cpython-36.pyc
index 378dbc6..956af70 100644
Binary files a/src/ChalkBox/__pycache__/views.cpython-36.pyc and b/src/ChalkBox/__pycache__/views.cpython-36.pyc differ
diff --git a/src/ChalkBox/__pycache__/wsgi.cpython-36.pyc b/src/ChalkBox/__pycache__/wsgi.cpython-36.pyc
index c8c33c4..58b27de 100644
Binary files a/src/ChalkBox/__pycache__/wsgi.cpython-36.pyc and b/src/ChalkBox/__pycache__/wsgi.cpython-36.pyc differ
diff --git a/src/accounts/__pycache__/__init__.cpython-36.pyc b/src/accounts/__pycache__/__init__.cpython-36.pyc
index da880a8..6b68310 100644
Binary files a/src/accounts/__pycache__/__init__.cpython-36.pyc and b/src/accounts/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/accounts/__pycache__/admin.cpython-36.pyc b/src/accounts/__pycache__/admin.cpython-36.pyc
index 57cea96..ee8a983 100644
Binary files a/src/accounts/__pycache__/admin.cpython-36.pyc and b/src/accounts/__pycache__/admin.cpython-36.pyc differ
diff --git a/src/accounts/__pycache__/models.cpython-36.pyc b/src/accounts/__pycache__/models.cpython-36.pyc
index 5390aca..4347fa0 100644
Binary files a/src/accounts/__pycache__/models.cpython-36.pyc and b/src/accounts/__pycache__/models.cpython-36.pyc differ
diff --git a/src/accounts/__pycache__/urls.cpython-36.pyc b/src/accounts/__pycache__/urls.cpython-36.pyc
index ab5e0d1..32ed767 100644
Binary files a/src/accounts/__pycache__/urls.cpython-36.pyc and b/src/accounts/__pycache__/urls.cpython-36.pyc differ
diff --git a/src/accounts/__pycache__/views.cpython-36.pyc b/src/accounts/__pycache__/views.cpython-36.pyc
index fd26d06..d7cd650 100644
Binary files a/src/accounts/__pycache__/views.cpython-36.pyc and b/src/accounts/__pycache__/views.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0001_initial.cpython-36.pyc b/src/accounts/migrations/__pycache__/0001_initial.cpython-36.pyc
index c4a8edf..1dae273 100644
Binary files a/src/accounts/migrations/__pycache__/0001_initial.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0001_initial.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0002_auto_20171104_1716.cpython-36.pyc b/src/accounts/migrations/__pycache__/0002_auto_20171104_1716.cpython-36.pyc
index 19927ef..27bfb8c 100644
Binary files a/src/accounts/migrations/__pycache__/0002_auto_20171104_1716.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0002_auto_20171104_1716.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0003_user_courses.cpython-36.pyc b/src/accounts/migrations/__pycache__/0003_user_courses.cpython-36.pyc
index 6dc649f..64a5fdb 100644
Binary files a/src/accounts/migrations/__pycache__/0003_user_courses.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0003_user_courses.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0004_user_phone_number.cpython-36.pyc b/src/accounts/migrations/__pycache__/0004_user_phone_number.cpython-36.pyc
index 2cc0b6e..634fb24 100644
Binary files a/src/accounts/migrations/__pycache__/0004_user_phone_number.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0004_user_phone_number.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0005_auto_20171120_1554.cpython-36.pyc b/src/accounts/migrations/__pycache__/0005_auto_20171120_1554.cpython-36.pyc
index cdd15ac..e14f564 100644
Binary files a/src/accounts/migrations/__pycache__/0005_auto_20171120_1554.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0005_auto_20171120_1554.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0006_user_biography.cpython-36.pyc b/src/accounts/migrations/__pycache__/0006_user_biography.cpython-36.pyc
index a964e9e..5dd5006 100644
Binary files a/src/accounts/migrations/__pycache__/0006_user_biography.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0006_user_biography.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0007_remove_user_phone_number.cpython-36.pyc b/src/accounts/migrations/__pycache__/0007_remove_user_phone_number.cpython-36.pyc
index f736e39..f9be411 100644
Binary files a/src/accounts/migrations/__pycache__/0007_remove_user_phone_number.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0007_remove_user_phone_number.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0008_user_phone_number.cpython-36.pyc b/src/accounts/migrations/__pycache__/0008_user_phone_number.cpython-36.pyc
index 6a968d8..053cab9 100644
Binary files a/src/accounts/migrations/__pycache__/0008_user_phone_number.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0008_user_phone_number.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0009_auto_20171120_1617.cpython-36.pyc b/src/accounts/migrations/__pycache__/0009_auto_20171120_1617.cpython-36.pyc
index ffe7f2d..d2e94bd 100644
Binary files a/src/accounts/migrations/__pycache__/0009_auto_20171120_1617.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0009_auto_20171120_1617.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/0010_auto_20171120_1619.cpython-36.pyc b/src/accounts/migrations/__pycache__/0010_auto_20171120_1619.cpython-36.pyc
index 3ca2668..226b283 100644
Binary files a/src/accounts/migrations/__pycache__/0010_auto_20171120_1619.cpython-36.pyc and b/src/accounts/migrations/__pycache__/0010_auto_20171120_1619.cpython-36.pyc differ
diff --git a/src/accounts/migrations/__pycache__/__init__.cpython-36.pyc b/src/accounts/migrations/__pycache__/__init__.cpython-36.pyc
index 31dd0f5..62f3ccd 100644
Binary files a/src/accounts/migrations/__pycache__/__init__.cpython-36.pyc and b/src/accounts/migrations/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/assignments/__pycache__/__init__.cpython-36.pyc b/src/assignments/__pycache__/__init__.cpython-36.pyc
index eab93df..0d40aee 100644
Binary files a/src/assignments/__pycache__/__init__.cpython-36.pyc and b/src/assignments/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/assignments/__pycache__/admin.cpython-36.pyc b/src/assignments/__pycache__/admin.cpython-36.pyc
index 9ed31b9..ee7a711 100644
Binary files a/src/assignments/__pycache__/admin.cpython-36.pyc and b/src/assignments/__pycache__/admin.cpython-36.pyc differ
diff --git a/src/assignments/__pycache__/forms.cpython-36.pyc b/src/assignments/__pycache__/forms.cpython-36.pyc
index 77a2415..99d0aae 100644
Binary files a/src/assignments/__pycache__/forms.cpython-36.pyc and b/src/assignments/__pycache__/forms.cpython-36.pyc differ
diff --git a/src/assignments/__pycache__/models.cpython-36.pyc b/src/assignments/__pycache__/models.cpython-36.pyc
index 31882db..3b8fc47 100644
Binary files a/src/assignments/__pycache__/models.cpython-36.pyc and b/src/assignments/__pycache__/models.cpython-36.pyc differ
diff --git a/src/assignments/__pycache__/urls.cpython-36.pyc b/src/assignments/__pycache__/urls.cpython-36.pyc
index 218f6d1..ead2e3c 100644
Binary files a/src/assignments/__pycache__/urls.cpython-36.pyc and b/src/assignments/__pycache__/urls.cpython-36.pyc differ
diff --git a/src/assignments/__pycache__/views.cpython-36.pyc b/src/assignments/__pycache__/views.cpython-36.pyc
index bd7d004..ff9ba7b 100644
Binary files a/src/assignments/__pycache__/views.cpython-36.pyc and b/src/assignments/__pycache__/views.cpython-36.pyc differ
diff --git a/src/assignments/migrations/__pycache__/0001_initial.cpython-36.pyc b/src/assignments/migrations/__pycache__/0001_initial.cpython-36.pyc
index e2f2ba9..0daa566 100644
Binary files a/src/assignments/migrations/__pycache__/0001_initial.cpython-36.pyc and b/src/assignments/migrations/__pycache__/0001_initial.cpython-36.pyc differ
diff --git a/src/assignments/migrations/__pycache__/0002_assignment_question.cpython-36.pyc b/src/assignments/migrations/__pycache__/0002_assignment_question.cpython-36.pyc
index 27b5245..2e11b1d 100644
Binary files a/src/assignments/migrations/__pycache__/0002_assignment_question.cpython-36.pyc and b/src/assignments/migrations/__pycache__/0002_assignment_question.cpython-36.pyc differ
diff --git a/src/assignments/migrations/__pycache__/0003_auto_20171122_1433.cpython-36.pyc b/src/assignments/migrations/__pycache__/0003_auto_20171122_1433.cpython-36.pyc
index c1d19aa..3af7aba 100644
Binary files a/src/assignments/migrations/__pycache__/0003_auto_20171122_1433.cpython-36.pyc and b/src/assignments/migrations/__pycache__/0003_auto_20171122_1433.cpython-36.pyc differ
diff --git a/src/assignments/migrations/__pycache__/0004_auto_20171130_0213.cpython-36.pyc b/src/assignments/migrations/__pycache__/0004_auto_20171130_0213.cpython-36.pyc
index 7b5b166..d136ca9 100644
Binary files a/src/assignments/migrations/__pycache__/0004_auto_20171130_0213.cpython-36.pyc and b/src/assignments/migrations/__pycache__/0004_auto_20171130_0213.cpython-36.pyc differ
diff --git a/src/assignments/migrations/__pycache__/__init__.cpython-36.pyc b/src/assignments/migrations/__pycache__/__init__.cpython-36.pyc
index 81747e5..8d415e4 100644
Binary files a/src/assignments/migrations/__pycache__/__init__.cpython-36.pyc and b/src/assignments/migrations/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/assignments/urls.py b/src/assignments/urls.py
index 34c8164..3bbf688 100644
--- a/src/assignments/urls.py
+++ b/src/assignments/urls.py
@@ -1,10 +1,10 @@
 from django.conf.urls import url
-from .views import assignment_home, assignment_detail, assignment_submitted, assignment_edit
+from .views import assignment_home, assignment_detail, assignment_submitted, assignment_edit, assignment_create
 
 urlpatterns = [
 	url(r'^$', assignment_home, name="assignment_home"),
 	url(r'^(?P<aid>\d+)/$', assignment_detail, name="assignment_detail"),
 	url(r'^(?P<aid>\d+)/edit/$', assignment_edit, name="assignment_edit"),
 	url(r'^submitted/$', assignment_submitted, name="assignment_list"),
-#	url(r'^(?P<aid>\d+)/$', add_assignment, name="add_assignment"),
+	url(r'^create/$', assignment_create, name="assignment_create"),
 ]
diff --git a/src/assignments/views.py b/src/assignments/views.py
index 5d21fd6..01b37f4 100644
--- a/src/assignments/views.py
+++ b/src/assignments/views.py
@@ -64,3 +64,20 @@ def assignment_submitted(request, cid=None):
 		"id": course.id,
 	}
 	return render(request, "assignment/assignments_submitted.html", context)
+
+def assignment_create(request, cid=None):
+	if request.method=="POST":
+		form = AssignmentForm(request.POST or None, request.FILES or None)
+		if form.is_valid():
+			instance = form.save()
+			course = Course.objects.get(id=cid)
+			course.assignments.add(instance)
+			course.save()
+			return redirect("courses:assignments:assignment_home", cid=cid)
+	else: 
+		form = AssignmentForm(None)
+	context = {
+		"form": form,
+		"id": cid,
+	}
+	return render(request, "assignment/assignment_create.html", context)
diff --git a/src/courses/__pycache__/__init__.cpython-36.pyc b/src/courses/__pycache__/__init__.cpython-36.pyc
index 13bf45c..67d2286 100644
Binary files a/src/courses/__pycache__/__init__.cpython-36.pyc and b/src/courses/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/courses/__pycache__/admin.cpython-36.pyc b/src/courses/__pycache__/admin.cpython-36.pyc
index 964645d..6a2153d 100644
Binary files a/src/courses/__pycache__/admin.cpython-36.pyc and b/src/courses/__pycache__/admin.cpython-36.pyc differ
diff --git a/src/courses/__pycache__/models.cpython-36.pyc b/src/courses/__pycache__/models.cpython-36.pyc
index 5d70d57..0a21dc0 100644
Binary files a/src/courses/__pycache__/models.cpython-36.pyc and b/src/courses/__pycache__/models.cpython-36.pyc differ
diff --git a/src/courses/__pycache__/urls.cpython-36.pyc b/src/courses/__pycache__/urls.cpython-36.pyc
index 71b4dac..9d682c1 100644
Binary files a/src/courses/__pycache__/urls.cpython-36.pyc and b/src/courses/__pycache__/urls.cpython-36.pyc differ
diff --git a/src/courses/__pycache__/views.cpython-36.pyc b/src/courses/__pycache__/views.cpython-36.pyc
index fa7b5b0..001e198 100644
Binary files a/src/courses/__pycache__/views.cpython-36.pyc and b/src/courses/__pycache__/views.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0001_initial.cpython-36.pyc b/src/courses/migrations/__pycache__/0001_initial.cpython-36.pyc
index 32b1865..4eedc91 100644
Binary files a/src/courses/migrations/__pycache__/0001_initial.cpython-36.pyc and b/src/courses/migrations/__pycache__/0001_initial.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0002_auto_20171103_1158.cpython-36.pyc b/src/courses/migrations/__pycache__/0002_auto_20171103_1158.cpython-36.pyc
index 77260f6..fb29a61 100644
Binary files a/src/courses/migrations/__pycache__/0002_auto_20171103_1158.cpython-36.pyc and b/src/courses/migrations/__pycache__/0002_auto_20171103_1158.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0003_auto_20171103_1520.cpython-36.pyc b/src/courses/migrations/__pycache__/0003_auto_20171103_1520.cpython-36.pyc
index 2eb7af6..8353bae 100644
Binary files a/src/courses/migrations/__pycache__/0003_auto_20171103_1520.cpython-36.pyc and b/src/courses/migrations/__pycache__/0003_auto_20171103_1520.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0004_remove_course_user.cpython-36.pyc b/src/courses/migrations/__pycache__/0004_remove_course_user.cpython-36.pyc
index f83c535..ad83679 100644
Binary files a/src/courses/migrations/__pycache__/0004_remove_course_user.cpython-36.pyc and b/src/courses/migrations/__pycache__/0004_remove_course_user.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0005_auto_20171104_1853.cpython-36.pyc b/src/courses/migrations/__pycache__/0005_auto_20171104_1853.cpython-36.pyc
index cd9896b..ffbc87e 100644
Binary files a/src/courses/migrations/__pycache__/0005_auto_20171104_1853.cpython-36.pyc and b/src/courses/migrations/__pycache__/0005_auto_20171104_1853.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0006_course_syllabus.cpython-36.pyc b/src/courses/migrations/__pycache__/0006_course_syllabus.cpython-36.pyc
index e34a356..1068f19 100644
Binary files a/src/courses/migrations/__pycache__/0006_course_syllabus.cpython-36.pyc and b/src/courses/migrations/__pycache__/0006_course_syllabus.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0007_course_assignments.cpython-36.pyc b/src/courses/migrations/__pycache__/0007_course_assignments.cpython-36.pyc
index adb4005..ebd2301 100644
Binary files a/src/courses/migrations/__pycache__/0007_course_assignments.cpython-36.pyc and b/src/courses/migrations/__pycache__/0007_course_assignments.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0008_auto_20171127_1339.cpython-36.pyc b/src/courses/migrations/__pycache__/0008_auto_20171127_1339.cpython-36.pyc
index d187b4f..2d8c1d1 100644
Binary files a/src/courses/migrations/__pycache__/0008_auto_20171127_1339.cpython-36.pyc and b/src/courses/migrations/__pycache__/0008_auto_20171127_1339.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0009_auto_20171127_1414.cpython-36.pyc b/src/courses/migrations/__pycache__/0009_auto_20171127_1414.cpython-36.pyc
index 52dc701..c0af9fb 100644
Binary files a/src/courses/migrations/__pycache__/0009_auto_20171127_1414.cpython-36.pyc and b/src/courses/migrations/__pycache__/0009_auto_20171127_1414.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/0010_auto_20171130_0213.cpython-36.pyc b/src/courses/migrations/__pycache__/0010_auto_20171130_0213.cpython-36.pyc
index 9b4efc4..1b7adc1 100644
Binary files a/src/courses/migrations/__pycache__/0010_auto_20171130_0213.cpython-36.pyc and b/src/courses/migrations/__pycache__/0010_auto_20171130_0213.cpython-36.pyc differ
diff --git a/src/courses/migrations/__pycache__/__init__.cpython-36.pyc b/src/courses/migrations/__pycache__/__init__.cpython-36.pyc
index ef6f8c1..2f665b2 100644
Binary files a/src/courses/migrations/__pycache__/__init__.cpython-36.pyc and b/src/courses/migrations/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/db.sqlite3 b/src/db.sqlite3
index 2c1ece7..1eebfbd 100644
Binary files a/src/db.sqlite3 and b/src/db.sqlite3 differ
diff --git a/src/submissions/__pycache__/__init__.cpython-36.pyc b/src/submissions/__pycache__/__init__.cpython-36.pyc
index 3bbe7af..b1b840a 100644
Binary files a/src/submissions/__pycache__/__init__.cpython-36.pyc and b/src/submissions/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/submissions/__pycache__/admin.cpython-36.pyc b/src/submissions/__pycache__/admin.cpython-36.pyc
index 863ac08..baf5bf6 100644
Binary files a/src/submissions/__pycache__/admin.cpython-36.pyc and b/src/submissions/__pycache__/admin.cpython-36.pyc differ
diff --git a/src/submissions/__pycache__/forms.cpython-36.pyc b/src/submissions/__pycache__/forms.cpython-36.pyc
index 7ff7bcc..22c247e 100644
Binary files a/src/submissions/__pycache__/forms.cpython-36.pyc and b/src/submissions/__pycache__/forms.cpython-36.pyc differ
diff --git a/src/submissions/__pycache__/models.cpython-36.pyc b/src/submissions/__pycache__/models.cpython-36.pyc
index 3fe853d..0de53ce 100644
Binary files a/src/submissions/__pycache__/models.cpython-36.pyc and b/src/submissions/__pycache__/models.cpython-36.pyc differ
diff --git a/src/submissions/__pycache__/urls.cpython-36.pyc b/src/submissions/__pycache__/urls.cpython-36.pyc
index 05128b0..316db7b 100644
Binary files a/src/submissions/__pycache__/urls.cpython-36.pyc and b/src/submissions/__pycache__/urls.cpython-36.pyc differ
diff --git a/src/submissions/__pycache__/views.cpython-36.pyc b/src/submissions/__pycache__/views.cpython-36.pyc
index dc35b50..15d8e41 100644
Binary files a/src/submissions/__pycache__/views.cpython-36.pyc and b/src/submissions/__pycache__/views.cpython-36.pyc differ
diff --git a/src/submissions/migrations/__pycache__/0001_initial.cpython-36.pyc b/src/submissions/migrations/__pycache__/0001_initial.cpython-36.pyc
index aa49570..4c03be4 100644
Binary files a/src/submissions/migrations/__pycache__/0001_initial.cpython-36.pyc and b/src/submissions/migrations/__pycache__/0001_initial.cpython-36.pyc differ
diff --git a/src/submissions/migrations/__pycache__/0002_submission_sumission.cpython-36.pyc b/src/submissions/migrations/__pycache__/0002_submission_sumission.cpython-36.pyc
index 8619c1e..e9ad139 100644
Binary files a/src/submissions/migrations/__pycache__/0002_submission_sumission.cpython-36.pyc and b/src/submissions/migrations/__pycache__/0002_submission_sumission.cpython-36.pyc differ
diff --git a/src/submissions/migrations/__pycache__/0003_auto_20171120_1805.cpython-36.pyc b/src/submissions/migrations/__pycache__/0003_auto_20171120_1805.cpython-36.pyc
index d211c89..b8bc77e 100644
Binary files a/src/submissions/migrations/__pycache__/0003_auto_20171120_1805.cpython-36.pyc and b/src/submissions/migrations/__pycache__/0003_auto_20171120_1805.cpython-36.pyc differ
diff --git a/src/submissions/migrations/__pycache__/0004_auto_20171122_1433.cpython-36.pyc b/src/submissions/migrations/__pycache__/0004_auto_20171122_1433.cpython-36.pyc
index 458d1a5..f926759 100644
Binary files a/src/submissions/migrations/__pycache__/0004_auto_20171122_1433.cpython-36.pyc and b/src/submissions/migrations/__pycache__/0004_auto_20171122_1433.cpython-36.pyc differ
diff --git a/src/submissions/migrations/__pycache__/__init__.cpython-36.pyc b/src/submissions/migrations/__pycache__/__init__.cpython-36.pyc
index ba1239a..1d83a13 100644
Binary files a/src/submissions/migrations/__pycache__/__init__.cpython-36.pyc and b/src/submissions/migrations/__pycache__/__init__.cpython-36.pyc differ
diff --git a/src/templates/assignment/assignments_instructor.html b/src/templates/assignment/assignments_instructor.html
index bbbd3da..ab7d026 100644
--- a/src/templates/assignment/assignments_instructor.html
+++ b/src/templates/assignment/assignments_instructor.html
@@ -55,7 +55,7 @@
 	       	{% endfor %}
        	</tbody>
 	</table>
-	
+	<a href='{%url "courses:assignments:assignment_create" cid=id %}'><button type="submit" class="btn btn-default"><i class="fa fa-plus" aria-hidden="true"></i> Create Assignment</button></a>
     </main>
   </div>
 </div>
